docs/modules/ROOT/pages/index.adoc

```diff
@@ -28,7 +28,8 @@ It is the default developer interface for both Enterprise and Community editions
 It comes out-of-the-box with all of Neo4j’s graph database offerings, including Neo4j Server (Community and Enterprise editions) and Neo4j Desktop (all OS versions).
 
 Neo4j Browser is perfect for running ad-hoc graph queries, with just enough ability to prototype a Neo4j-based application.
```

Comment:
```suggestion
Neo4j Browser is suitable for running ad-hoc graph queries, with the appropriate ability to prototype a Neo4j-based application.
```
I know this is out of the scope of this PR, but this makes the text a bit less marketingy.